moped-database/views/project_list_view.sql

```diff
@@ -219,10 +219,10 @@ LEFT JOIN project_district_association districts ON mp.project_id = districts.pr
 LEFT JOIN LATERAL (
     SELECT
         mpn.project_note,
-        mpn.date_created
+        mpn.created_at AS date_created
     FROM moped_proj_notes mpn
     WHERE mpn.project_id = mp.project_id AND mpn.project_note_type = 2 AND mpn.is_deleted = false
-    ORDER BY mpn.date_created DESC
+    ORDER BY mpn.created_at DESC
```

Comment:
these changes appeared after i merged main into the branch, the github workflow changed it

Reply:
this change surprised me because I would expect it to have already been applied to `main` and not be introducing a diff unrelated to this PR. I think this is just the GH workflow catching up to previous changes, though. 🤔

Reply:
this confuses me too!! was the project list view broken at any point due to this?

Reply:
ok it wasn't, that is a relief. very happy with this workflow!